packages/cozy-client/src/types.js

```diff
@@ -305,6 +305,8 @@ import { QueryDefinition } from './queries/dsl'
  * @property {string} name - Name of the folder
  * @property {object} metadata - Metadata of the folder
  * @property {object} type - Type of the folder
+ * @property {object} attributes - Folder attributes
+ * @property {string} attributes.path - Folder path
```

Comment:
Since other attributes are not within the attributes object, can you do the same here? And update your code please? (cf https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.files.md#iocozyfiles path should be at the same "level" as name) 

We have a duplication issue of the attributes (cf https://github.com/cozy/cozy-client/issues/1118) but let's keep the coherence.

Reply:
Ok, I then also removed the use  of ".attibutes" in the folder.js file